### PR TITLE
UPDATE MENTOR.YML

### PR DESCRIPTION
**Pull request by ShivamRai2003**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Comment by ShivamRai2003** [comment]
MENTORS PLEASE REVIEWE IT AND MERGE IT PLZZ

File: _data/mentors.yml [hunk 1]
```diff
@@ -10,6 +10,11 @@
 #   lat: Mentor's Latitude
 #   lng: Mentor's Longitude
 
+- name: Sean McGregor
+  github: Sean McGregor
```
Comment (nikit19) [inline _data/mentors.yml]: This github account doesn't exist. Probably there is no space

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make this change

**Comment by ShivamRai2003** [comment]
please see this one it is correct

**Review by isuruAb — COMMENTED** [review]
please squash your commits

**Pull request closed without merging**